pages/docs/tutorials/competitive-programming.md

```diff
@@ -171,14 +171,14 @@ fun main() {
 
 </div>
 
-In this dense code, in addition to collection transformations, you can see such handy Kotlin features as local functions,
-and [elvis operator](/docs/reference/null-safety.html#elvis-operator) `?:`
-that allow to express the 
-[idioms](/docs/reference/idioms.html) like "take value if it is positive or else use length" with concise and readable 
-expression like `.takeIf { it >= 0 } ?: s.length`, yet it is perfectly fine with Kotlin to create additional mutable
+In this dense code, in addition to collection transformations, you can see such handy Kotlin features as local functions
+and the [elvis operator](/docs/reference/null-safety.html#elvis-operator) `?:`
+that allow to express 
+[idioms](/docs/reference/idioms.html) like "take the value if it is positive or else use length" with concise and readable 
+expressions like `.takeIf { it >= 0 } ?: s.length`, yet it is perfectly fine with Kotlin to create additional mutable
```

Comment:
```suggestion
expression like `.takeIf { it >= 0 } ?: s.length`, yet it is perfectly fine with Kotlin to create additional mutable
```

Reply:
The original version had the noun in the singular form without an article, and you suggest to revert it. Why?

Reply:
Strange thing, on the **Files changed** tab GitHub shows me this part like: `with a concise and readable expressions`. I've checked in your branch and there is no article, so everything is fine.
Thanks!